recipes/arm_openmpi_install.rb

```diff
@@ -0,0 +1,46 @@
+
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: aws-parallelcluster
+# Recipe:: arm_openmpi_install
+#
+# Copyright 2013-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+return if node['conditions']['ami_bootstrapped'] || !arm_instance?
```

Comment:
if there are no other place from where `arm_openmpi_install` is included, then `if node['conditions']['ami_bootstrapped']` can be removed, because it's already included in the caller

Reply:
Makes sense.